src/main/scala/za/co/absa/hyperdrive/trigger/persistance/DagInstanceRepository.scala

```diff
@@ -51,6 +53,17 @@ class DagInstanceRepositoryImpl extends DagInstanceRepository {
     }.transactionally
   ).map(_ => (): Unit)
 
+  override def insertJoinedDagInstances(dagInstancesJoined: Seq[DagInstanceJoined])(implicit executionContext: ExecutionContext): Future[Unit] = db.run(
+    DBIO.sequence {
+      dagInstancesJoined.map { dagInstanceJoined =>
+        for {
```

Comment:
Please extract this duplicated code fragment

Reply:
Ok